core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/UpcheckHandler.java

```diff
@@ -19,8 +19,23 @@
 import io.vertx.ext.web.RoutingContext;
 
 public class UpcheckHandler implements Handler<RoutingContext> {
+
+  private final boolean failToLoad;
```

Comment:
As above, failedToLoadAnySigners

Reply:
If the service is up but the signers failed to load, is there any mitigation via RPC or is the only recourse to restart the service?

Reply:
I think with this we just have a way to tell if w3s fails to load signers from one of the sources and if yes, we can restart it.